### PR TITLE
Optimize Coog image size

### PR DESCRIPTION
**Pull request by alikefia**
Mount is disabled (script to be reviewed since no bash on alpine)

### CODE REVIEW

File: images/coog/py3/Dockerfile [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
-        libpq-dev \
-        libldap2-dev \
-        libsasl2-dev \
-        redis-tools \
-        graphviz \
-        cifs-utils \
-    && apt-get clean && rm -rf /var/lib/apt/lists/*
-
-RUN pip install \
+RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories \
```
Comment (JCavallo) [inline images/coog/py3/Dockerfile]: stable

**Comment by alikefia** [comment]
750Mo => 350Mo

File: images/coog/py3/Dockerfile (hunk 1 above)
Comment (alikefia) [reply images/coog/py3/Dockerfile]: We will wait until alpine-3.8
Mais au moins les tests sont fait

**Comment by alikefia** [comment]
Merged to #44

**Pull request closed without merging**